packages/react-components/react-positioning/src/types.ts

```diff
@@ -173,21 +170,29 @@ export interface PositioningOptions {
    * @default true
    */
   useTransform?: boolean;
+
+  /**
+   * If false, does not position anything
+   */
+  enabled?: boolean;
 }
 
 export interface PositioningProps
   extends Pick<
     PositioningOptions,
     | 'align'
-    | 'flipBoundary'
-    | 'overflowBoundary'
-    | 'overflowBoundaryPadding'
-    | 'position'
-    | 'offset'
     | 'arrowPadding'
     | 'autoSize'
     | 'coverTarget'
+    | 'enabled'
```

Comment:
```suggestion
```

Reply:
Looks like I misunderstood - I thought we would like to put the `enabled` here and get rid of:
```
interface UsePositioningOptions extends PositioningProps {
  /**
   * If false, does not position anything
   */
  enabled?: boolean;
}
```

Reply:
Got it :) removing enabled